models/artifact_helpers.go

```diff
@@ -44,6 +47,32 @@ func FindArtifacts(q *reform.Querier) ([]*Artifact, error) {
 	return artifacts, nil
 }
 
+// FindArtifactsByIDs finds artifacts by IDs.
+func FindArtifactsByIDs(q *reform.Querier, ids []string) (map[string]*Artifact, error) {
```

Comment:
shouldn't be consistent and return just array instead of map?

Reply:
it's easier to operate when using the map, plus there are other functions with a similar approach.